index.html

```diff
@@ -0,0 +1,203 @@
+                    <div class="wrap-card">
+                        <h3 id="article-h">Multi-Post Stories</h3>
+                        <p id="featured-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                        <div class="lang">
+                            <button type="button">css</button>
+                            <button type="button">html</button>
+                            <button type="button">bootstrap</button>
+                            <button type="button">Ruby</button>
+                        </div>
+                    </div>
+                    <div>
+                        <a class="see-project" href="#">See Project</a>
+                    </div>
+                </article>
+                <article class="project1">
+                    <div class="wrap-card">
+                        <h3>Profesional Art Printing Data</h3>
+                        <p>A daily selection of privately personalized reads;  no accounts or sign-ups required. has been the industry's standard</p>
+                        <div class="lang">
+                            <button type="button">html</button>
+                            <button type="button">bootstrap</button>
+                            <button type="button">Ruby</button>
+                        </div>
+                    </div>
+                    <div class="card-link">
+                        <a class="see-project" href="#">See Project</a>
+                    </div>
+                </article>
+                <article class="project2">
+                    <div class="wrap-card">
+                        <h3>Data Dashboard HealthcarePro</h3>
+                        <p>Tempore quia aspernatur exercitationem quo libero? Quod cupiditate</p>
+                        <div class="lang">
+                            <button type="button">html</button>
+                            <button type="button">bootstrap</button>
+                            <button type="button">Ruby</button>
+                        </div>
+                    </div>
+                    <div class="card-link">
+                        <a class="see-project" href="#">See Project</a>
+                    </div>
+                </article>
+                <article class="project3">
+                    <div class="wrap-card">
+                        <h3>Website Protfolio</h3>
+                        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore quia aspernatur exercitationem quo libero?</p>
+                        <div class="lang">
+                            <button type="button">html</button>
+                            <button type="button">bootstrap</button>
+                            <button type="button">Ruby</button>
+                        </div>
```

Comment:
- [ ] [REQUIRED] The list of project technologies should(e.g., "JavasScript", "HTML" and "CSS") a list `ul` and each of them a `li`. Please make changes!